tigerpath/majors_and_certificates/majors/NEU_2018.json

```
@@ -0,0 +1,348 @@
+                                "NEU 475: Basal Ganglia in Health and Disease",
+                                "NEU 501A: Molecules to Systems"
+                            ]
+                        },
+                        {
+                            "name": "Neural Computation",
+                            "max_counted": 5,
+                            "min_needed": null,
+                            "explanation": "",
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "PSY/NEU 330: Connectionist Modeling",
+                                "PSY/NEU 338: Animal Learning & Decision-making",
+                                "PSY/NEU 340: Neuroeconomics",
+                                "NEU 437: Computational Neuroscience",
+                                "NEU 457: Measurement and Analysis of Neural Circuit Dynamics",
+                                "COS 402: Artificial Intelligence",
+                                "COS 324: Fundamentals of Machine Learning",
+                                "COS 485: Neural Networks: Theory and Applications",
+                                "ELE 364: Machine Learning for Predictive Data Analytics",
+                                "MAT/APC 323: Topics in Mathematical Modeling - Mathematical Neuroscience"
+                            ]
+                        },
+                        {
+                            "name": "Systems & Circuits",
+                            "max_counted": 5,
+                            "min_needed": null,
+                            "explanation": "",
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "PSY/NEU 260: Life Cycle of Behaviors",
+                                "PSY/NEU 316: Cognitive Neuroscience of Selective Attention",
+                                "NEU 301: Cellular Neurobiology",
+                                "PSY/NEU 336: Diversity of Brains",
+                                "PSY 345/NEU 325: Sensation & Perception",
+                                "NEU/PSY 413: The Neurobiology of Stress",
+                                "NEU 200: Functional Neuroanatomy",
+                                "NEU/PSY 422: Neural Dynamics of Cognition",
+                                "NEU 427: Systems Neuroscience",
+                                "NEU 457: Measurement and Analysis of Neural Circuit Dynamics",
+                                "NEU 475: Basal Ganglia in Health and Disease",
+                                "NEU 501A: Molecules to Systems",
+                                "NEU 502A: From Molecules to Systems to Behavior"
+                            ]
+                        },
+                        {
+                            "name": "Social & Cognitive Neuroscience",
+                            "max_counted": 5,

```

Maybe this should be a 3, so that they're forced to take the other two courses from another subject area?
```suggestion
                            "max_counted": 3,
```